src/test/HostActivationTests/FrameworkResolution/RollForwardOnNoCandidateFxSettings.cs

```diff
@@ -198,6 +198,74 @@ public void NoInheritance_MoreRestrictive(SettingLocation settingLocation, strin
                 resolvedFramework);
         }
 
+        [Theory]
+        [InlineData("1.0.0", "2.5.5", "2.5.5")]
+        [InlineData("2.0.0", "2.5.5", "2.5.5")]
+        [InlineData("2.5.4", "2.5.5", "2.5.5")]
+        [InlineData("2.5.5", "2.5.5", "2.5.5")]
+        [InlineData("2.5.5", "2.5.4", "2.5.4")]
+        [InlineData("2.5.5", "2.5.3", null)]
+        [InlineData("2.5.5", "2.5.5-preview1", null)]
+        public void FxVersionCLIOverride(string frameworkReferenceVersion, string fxVersion, string resolvedFramework)
```

Comment:
Maybe I'm missing something, but I'm not sure how these are related to `RollForwardOnNoCandidateFxSettings`? They seem to just be testing setting fx version?

Reply:
Yeah … I was kind of torn … it made it easy :-)…
I moved them out into a separate class. There is still overlap, but much smaller.